pcdsdevices/sqr1.py

```diff
@@ -343,6 +348,19 @@ def multi_axis_move(
         >>> tri_sphere = SQR1(prefix="SQR1:", name="tri_sphere")
         >>> status = tri_sphere.multi_axis_move(x_sp=1.0, y_sp=2.0, z_sp=3.0)
         """
+
+        if preset_pos is not None:
+            axis_list = [self.x, self.y, self.z, self.rz, self.ry, self.rz]
+            if any(hasattr(a.presets.positions, preset_pos) for a in axis_list):
+                x_sp = (getattr(self.x.presets.positions, preset_pos)).pos
+                y_sp = (getattr(self.y.presets.positions, preset_pos)).pos
+                z_sp = (getattr(self.z.presets.positions, preset_pos)).pos
+                rx_sp = (getattr(self.rx.presets.positions, preset_pos)).pos
+                ry_sp = (getattr(self.ry.presets.positions, preset_pos)).pos
+                rz_sp = (getattr(self.rz.presets.positions, preset_pos)).pos
+            else:
+                logger.error('One of the axes is missing the desired state.')
```

Comment:
should this `return` or `raise` as well in the error state to prevent the rest of the function from running?